modules/ModuleBuilder/parsers/views/AbstractMetaDataImplementation.php

```diff
@@ -344,16 +344,6 @@ protected function _saveToFile($filename, $defs, $useVariables = true, $forPopup
 
         $out .= ";\n";
 
-        if(!empty($this->_originalViewTemplateDefs)) {
```

Comment:
The listviewdefs.php depends on this code. In 7.9, I added a means to customise the templateMeta in the list view, in the same way you can in the edit / detail view. So that you can include javascript, and customise the bulk action button, and add your own custom buttons.

When you save a list view layout in studio this will cause the list view buttons to disappear. For example the list view in the emails module.

See #3496 for details.

So ... We need to find an other way to solve this issue.

Reply:
Ok I think this will do. In the case of the listview the structure of the defs is very different and does not contain an array index with the name of the view.

Reply:
@daniel-samson please review these changes

Reply:
Just test your new changes it seems to work